core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7139,6 +7163,12 @@ public static final class Name {
         "alluxio.master.metadata.sync.ufs.prefetch.timeout";
     public static final String MASTER_METASTORE = "alluxio.master.metastore";
     public static final String MASTER_METASTORE_DIR = "alluxio.master.metastore.dir";
+    public static final String MASTER_METASTORE_ROCKS_PARALLEL_BACKUP =
+        "alluxio.master.parallel.backup.rocksdb";
+    public static final String MASTER_METASTORE_ROCKS_PARALLEL_BACKUP_COMPRESSION_LEVEL =
+        "alluxio.master.parallel.backup.rocksdb.compression.level";
+    public static final String MASTER_METASTORE_ROCKS_PARALLEL_BACKUP_THREADS =
+        "alluxio.master.parallel.backup.rocksdb.thread.pool.size";
```

Comment:
Let's make the property name match the variable
```suggestion
    public static final String MASTER_METASTORE_ROCKS_PARALLEL_BACKUP =
        "alluxio.master.metastore.rocks.parallel.backup";
    public static final String MASTER_METASTORE_ROCKS_PARALLEL_BACKUP_COMPRESSION_LEVEL =
        "alluxio.master.metastore.rocks.parallel.backup.compression.level";
    public static final String MASTER_METASTORE_ROCKS_PARALLEL_BACKUP_THREADS =
        "alluxio.master.metastore.rocks.parallel.backup.threads";
```